libs/ui/src/components/NeoNftCard/NeoNftCard.vue

```diff
@@ -63,6 +64,18 @@
         </div>
       </div>
     </a>
+
+    <template v-else>
+      <div class="media-object nft-media">
+        <div class="is-square image">
+          <NeoSkeleton :rounded="false" full-size no-margin />
+        </div>
+      </div>
+      <div :class="`nft-media-info__${variant}`">
+        <NeoSkeleton size="medium" no-margin />
+        <NeoSkeleton size="small" no-margin width="100px" />
+      </div>
+    </template>
```

Comment:
This thing can be moved in `DynamicGrid`?

Reply:
> But this would mean that I need to include `NeoNftCard.scss` in `<ItemGrid />` (although it seems like an unrelated component). Is this acceptable?

Okay, let's move on